openapi.yml

```diff
@@ -366,95 +349,20 @@ paths:
         405:
           description: Method not allowed
-  /threads/counts:
-    get:
-      tags:
-        - Threads
-        - Existing
-      summary: Get count of threads for a user
-      description: Return count of threads for a user depending on a number of filters. This endpoint will be used for response operations and returns a number of counts which can be seen below in the responses
-      parameters:
-        - name: user_id
-          in: query
-          description: User UUID assigned to the threads
-          schema:
-            type: string
-            format: uuid
-            example: 6934f0c7-b595-435b-aa8f-291d6b6070d3
-        - name: unread
-          in: query
-          description: Whether to get a count of the unread threads
-          schema:
-            type: bool
-            default: false
-        - name: is_closed
-          in: query
-          description: True | False is the thread closed
-          schema:
-            type: bool
-            default: false
-        - name: survey
-          in: query
-          description: UUID of survey that the threads pertain to
-          schema:
-            type: string
-            format: uuid
-            example: 85e75391-9b91-4322-8984-3d53da29186e
-        - name: ru_ref
-          in: query
-          description: Business reference
-          schema:
-            type: string
-            example: 41104403458
-        - name: category
-          in: query
-          description: A string referring to what type a message is
-          schema:
-            type: string
-            enum:
-              - SURVEY
-              - TECHNICAL
-              - MISC
-            default: SURVEY
-        - name: case_id
-          in: query
-          description: UUID referring to the case
-          schema:
-            type: string
-            format: uuid
-            example: 6934f0c7-b595-435b-aa8f-291d6b6070d3
-      responses:
-        200:
-          description: Successfully returned thread count
-          content:
-            application/json:
-              schema:
-                type: object
-                properties:
-                  totals:
-                    type: array
-                    example:
-                      open: 2
-                      closed: 0
-                      my_conversations: 1
-                      initial_conversations: 2
-        400:
-          description: Invalid `user_id`
   /threads/count:
     get:
       tags:
         - Threads
         - Existing
-      summary: Get count of unread threads for a respondent
+      summary: Get count of unread threads for a user
       description: Return count of threads for a frontstage user. This is different from the above /threads/counts as this one only returns one count, specifically of unread messages linked to the user id, and is much more restricted as no parameters other than the `user_id` can be passed to filter. Only used for frontstage
```

Comment:
Probably doesn't need this explanation any more, at the very least it's now inaccurate.